modules/ROOT/pages/manage-versions-instances-concept.adoc

```diff
@@ -3,9 +3,9 @@ ifndef::env-site,env-github[]
 include::_attributes.adoc[]
 endif::[]
 
-In Anypoint Exchange (Exchange), you can add, delete, and deprecate API versions, also known as _Exchange assets_. An API version in Exchange can have its own RAML file, API portal, and URL. See xref:exchange::manage-versions.adoc[Manage Versions] for more information on how to manage API versions in exchange.
+In Anypoint Exchange (Exchange), you can add, delete, and deprecate API versions, also known as _Exchange assets_. An API version in Exchange can have its own RAML file, API portal, and URL. For more information on how to manage API versions in exchange, see xref:exchange::manage-versions.adoc[Manage Versions].
```

Comment:
For more information on > For more information about (to follow Kim's edits) (here and in the next paragraph)

exchange > Exchange (here and in the next paragraph)